modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/admin/throttlingpolicy/GetThrottlingPoliciesTestCase.java

```diff
@@ -0,0 +1,154 @@
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.am.integration.clients.admin.api.dto.ThrottlePolicyDetailsDTO;
+import org.wso2.am.integration.clients.admin.api.dto.ThrottlePolicyDetailsListDTO;
+import org.wso2.am.integration.test.helpers.AdminApiTestHelper;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationBaseTest;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationConstants;
+import org.wso2.am.integration.test.utils.http.HTTPSClientUtils;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.integration.common.admin.client.UserManagementClient;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.wso2.am.integration.test.utils.generic.APIMTestCaseUtils.encodeCredentials;
+
+public class GetThrottlingPoliciesTestCase extends APIMIntegrationBaseTest {
+    private final String getThrottlePoliciesResource = "/throttling/policies/search";
+    private final  String appPolicyName="50PerMin";
+    private final String subPolicyName ="Gold";
+    private final String apiPolicyName="10KPerMin";
+    private String getThrottlePoliciesUrl;
+
+    @Factory(dataProvider = "userModeDataProvider")
+    public GetThrottlingPoliciesTestCase(TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @DataProvider public static Object[][] userModeDataProvider() {
+        return new Object[][] { new Object[] { TestUserMode.SUPER_TENANT_ADMIN },
+                new Object[] { TestUserMode.TENANT_ADMIN } };
+    }
+
+    @BeforeClass(alwaysRun = true) public void setEnvironment() throws Exception {
+        super.init(userMode);
+        getThrottlePoliciesUrl =
+                adminURLHttps + APIMIntegrationConstants.REST_API_ADMIN_CONTEXT_FULL_0 + getThrottlePoliciesResource;
+    }
+
+    @Test(groups = { "wso2.am" }, description = "Get ThrottlePolicies List and check few default policies")
+    public void testThrottlePoliciesGet() throws Exception {
+        //construct get Throttle Policies url
+        URL getRequest = new URL(getThrottlePoliciesUrl);
+        ThrottlePolicyDetailsListDTO policies = getPolicies(user.getUserName(), user.getPassword(), getRequest);
+        List<ThrottlePolicyDetailsDTO> policyList = policies.getList();
+        Assert.assertTrue(containsPolicy(policyList, appPolicyName), "Doesn't contain app Policy");
+        Assert.assertTrue(containsPolicy(policyList, subPolicyName), "Doesn't contain sub Policy");
+        Assert.assertTrue(containsPolicy(policyList, apiPolicyName), "Doesn't contain api Policy");
+    }
+
+    /**
+     * Checks whether a certain policy exists in the policy list
```

Comment:
Empty line should be there after the method desciption

Reply:
Fixed with 3106d801961d2d8e82a5a35117427b8337a92455